packages/ui/input/radio/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import classnames from "classnames";
+import { bool, func, string } from "prop-types";
+
+import styles from "./index.css";
+
+
+export default function Radio({ checked, className, disabled, error, label, onChange, value: inputValue }) {
```

Comment:
@ailincaibianca destructure `{ ...other}` from the `props` object to the `<input` node. There are properties of importance to a `input type="radio"` to pass. Such as `name`.

Also, `id` sent to `<input` should not only be: `inputValue`.

Consider if the value is `1` for one `<Radio` but then another `<Radio` has the same value of `1` somewhere else in the app.

I would suggest we set it as either an explicitly passed `id` through props. or, if it is not passed, a combination of `name + value`:
```
 export default function Radio({ (...), id, name = "", ...others }) {
 const inputId = id ? id : `${name}-${inputValue}`;
 
```